lang/python/python3-sqlparse/Makefile

```diff
@@ -0,0 +1,37 @@
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=sqlparse
+PKG_VERSION:=0.3.0
+PKG_RELEASE:=1
+PKG_LICENSE:=BSD
```

Comment:
This is not correct SPDX License identifier. Take a look at our [CONTRIBUTING](https://github.com/openwrt/packages/blob/master/CONTRIBUTING.md) file or check [SPDX license identifiers](https://SPDX.org/licenses/).

Reply:
Also PKG_LICENSE_FILES can be added.

Reply:
Oh, I changed that.